src/schemas/json/devinit.schema-3.0.json

```diff
@@ -0,0 +1,130 @@
+{
+  "title": "Codespaces DevInit Configuration",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Can you make this draft-v4 so VS can parse it correctly?

Reply:
Should I also make `devinit.schema-1.0.json` be `draft-v4`?

Reply:
Yes, that would ensure best compatibility